frontend/src/app/app.routes.ts

```diff
@@ -156,6 +169,53 @@ export const routes: Routes = [
         ]
       },
       {
+        path: 'tasks',
+        component: TasksLayoutComponent,
+        children: [
+          { path: '', pathMatch: 'full', redirectTo: 'todo' },
+          {
+            path: 'todo',
+            component: TasksComponent,
+            data: {
+              taskListType: 'todo'
+            },
+            resolve: {
+              tasks: TasksResolverService
+            }
+          },
+          {
+            path: 'urgent',
+            component: TasksComponent,
+            data: {
+              taskListType: 'urgent'
+            },
+            resolve: {
+              tasks: TasksResolverService
+            }
+          },
+          {
+            path: 'mine',
+            component: TasksComponent,
+            data: {
+              taskListType: 'mine'
+            },
+            resolve: {
+              tasks: TasksResolverService
+            }
+          },
+          {
+            path: 'unassigned',
+            component: TasksComponent,
+            data: {
+              taskListType: 'unassigned'
+            },
+            resolve: {
+              tasks: TasksResolverService
+            }
+          }
+        ]
```

Comment:
Actually, there's something I would lose with params: there doesn't seem to be a way to have a root `/tasks` route that would redirect to `/tasks;tab=todo`, for example. So I think I'll leave it as is for the moment.